apps/transport/lib/transport_web/controllers/dataset_controller.ex

```diff
@@ -129,7 +129,8 @@ defmodule TransportWeb.DatasetController do
       data
       |> Enum.reject(fn %DB.DatasetScore{score: score} -> is_nil(score) end)
       |> Enum.group_by(fn %DB.DatasetScore{topic: topic} -> topic end)
-      |> Map.new(fn {topic, scores} -> {topic, scores |> List.last() |> DB.DatasetScore.score_for_humans()} end)
+      |> Enum.map(fn {topic, scores} -> {topic, scores |> List.last() |> DB.DatasetScore.score_for_humans()} end)
+      |> Enum.sort_by(fn {a, _b} -> a end)
```

Comment:
```suggestion
      |> Enum.sort_by(fn {topic, _scores} -> topic end)
```
pour conserver la lisibilité de la ligne précédente

Reply:
J'ai traité dans 18220d24de39f5bb59f45d765126b815cf1a7420 avec un changement un peu différent :
- c'est `_score` et plus `_scores` à ce stade
- et ça méritait un commentaire du coup

merci